### PR TITLE
Added more points in what's next section of Understanding Kubernetes …

### LINKED ISSUE #37714: Add more items in what's next section of Understanding Kubernetes objects
**This is a Bug Report**

**Problem:**
Below information is needed to mention in Kubernetes-objects [what's-next section](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#what-s-next)
- Learn more about deployments
- Learn about kubectl and kubectl commands

**Proposed Solution:**
Add Below points under Kubernetes-objects [what's-next section](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#what-s-next)
- Learn more about [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
- Learn about [kubectl](https://kubernetes.io/docs/reference/kubectl/) and [kubectl commands](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands)

**Page to Update:**
https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/

### PR DESCRIPTION
**Pull request by T-Lakshmi**
This PR adding 2 more points under what's next section of Understanding Kubernetes objects page.

Fixes #37714

### CODE REVIEW

**Review by divya-mohan0209 — COMMENTED** [review]
While these are fine, the phrasing Learn more about XYZ seems super repetitive especially with the new additions. Is it possible to rephrase them?

**Comment by T-Lakshmi** [comment]
Hi @divya-mohan0209,
How about this.

Learn more about the Following.
- Most important basic Kubernetes objects, such as Pod
- controllers in Kubernetes.
- Using the Kubernetes API explains some more API concepts.
- Deployments.
- kubectl and kubectl commands.

If this seems good then I will update it. WDYT?

**Comment by divya-mohan0209** [comment]
I like the idea. However, you'll need to rework the casing & the grammar for the suggestion.

**Comment by T-Lakshmi** [comment]
Sure, I will take care of it.

**Comment by T-Lakshmi** [comment]
Hi @divya-mohan0209 ,
I have updated the changes we discussed above.
Could you please take a look.

File: content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md [hunk 1]
```diff
@@ -115,7 +115,10 @@ detail the structure of that `.status` field, and its content for each different
 
 ## {{% heading "whatsnext" %}}
 
-* Learn about the most important basic Kubernetes objects, such as [Pod](/docs/concepts/workloads/pods/).
-* Learn about [controllers](/docs/concepts/architecture/controller/) in Kubernetes.
-* [Using the Kubernetes API](/docs/reference/using-api/) explains some more API concepts.
+Learn more about the following.
```
Comment (reylejano) [inline content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md:118]: nit
```suggestion
Learn more about the following:
```

Comment (T-Lakshmi) [reply content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md]: Hi @reylejano,
Thanks for your valuable suggestion.

i have incorporated the above suggestion.

**Comment by T-Lakshmi** [comment]
/assign @nate-double-u
Could you please approve PR.

**Review by nate-double-u — COMMENTED** [review]
Thanks for this @T-Lakshmi 

/approve

**Pull request merged**